### ISSUE TITLE
We shouldn't stop the production of Banshees against Phoenixes

### ISSUE THREAD

**Issue by RaphaelRoyerRivard** [RaphaelRoyerRivard/MicroMachine#915]
Vikings and Medivacs are even worse than Banshees against Phoenixes. We should have Marines that take care of them or otherwise we need to just ignore them until we can get Thors or BCs.

**Comment by Unusual229** [comment]
So you want to stop all air production until we can deal with the phoenixes? Sounds very abusable.

**Comment by RaphaelRoyerRivard** [comment]
No, I want to continue producing Banshees. Against TyrP's Phoenixes, our Banshees are the only air units that do enough to justify their cost.

**Comment by Unusual229** [comment]
You shouldn't be losing vikings, You need to control them as a group and keep them close together. With maybe like 4-5 vikings, you can 1 hit a phoenix (focus fire) and easily deal with them. Have marines close by to get rapid damage if the phoenix decide to dive.

**Comment by RaphaelRoyerRivard** [comment]
In the current situation, we have 1 Viking and there are 3-5 Phoenixes. We
also have 0 Marine as we use our Barracks to produce Marauders which are
the only unit appart from Banshees that are useful against Stalkers +
Phoenixes. You should take a look at the last game we lost against TyrP to
understand better the situation we want to improve.
